src/main/java/de/tum/cit/aet/artemis/iris/service/session/IrisCourseChatSessionService.java

```diff
@@ -134,13 +133,13 @@ protected void setLLMTokenUsageParameters(LLMTokenUsageService.LLMTokenUsageBuil
      */
     public void onJudgementOfLearningSet(CompetencyJol competencyJol) {
         var course = competencyJol.getCompetency().getCourse();
-        if (!irisSettingsService.isEnabledFor(IrisSubSettingsType.CHAT, course)) {
-            return;
-        }
+
+        // TODO: Add setting to activate/deactivate course chat for JOLs
+
```

Comment:
Yes, it will come in a follow-up PR. Currently there is an open PR for adding course chat settings, and the event is related to course chat settings.

See #9866